srcpkgs/mesa/template

```diff
@@ -147,21 +147,27 @@ else
 	configure_args+=" -Dgallium-vdpau=disabled -Dgallium-va=disabled -Dgallium-xvmc=disabled"
 fi
 
-# empty values introduced by leading comma are not allowed; the whole enumeration can be empty
-# _gallium_drivers is not affected by this problem
-_vulkan_drivers=${_vulkan_drivers/=,/=}
-_dri_drivers=${_dri_drivers/=,/=}
-
-configure_args+=" ${_gallium_drivers} ${_vulkan_drivers} ${_dri_drivers}"
-
```

Comment:
To minimize the diff, I'd leave this whole thing in and just remove `_gallium_drivers`